src/main/webapp/WEB-INF/templates/views/adminOnly/admin-project_create.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="fragments/head.html :: head"></head>
+<body>
+<div class="container-fluid">
+    <div class="container">
+        <div class="row" id="two-column-container">
+            <!-- PANEL -->
+            <div class="grid col-sm-12 ">
+                <!-- NAME -->
+                <div class="row justify-content-center regpadding rowinrightcol createuser">
+                    <h1 class="reglogheader">Create a new project</h1>
```

Comment:
az budem mat cas tuto triedu premenujem, len to bude vela suborov treba zmenit, ale nevadi, ono som myslela ze to bude len pre registraciu a login povodne